src/Chapter2.hs

```diff
@@ -620,7 +631,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven (a:b:rest) = a:takeEven rest
```

Comment:
Great pattern matching skills! 🔥 

By the way, as you don't use `b`, you can omit naming it and leave the wildcard instead 🙂 
```suggestion
takeEven (a:_:rest) = a : takeEven rest
```